pkg/oci/internal/signature/layer.go

```diff
@@ -29,10 +29,11 @@ import (
 )
 
 const (
-	sigkey    = "dev.cosignproject.cosign/signature"
-	certkey   = "dev.sigstore.cosign/certificate"
-	chainkey  = "dev.sigstore.cosign/chain"
-	BundleKey = "dev.sigstore.cosign/bundle"
+	sigkey       = "dev.cosignproject.cosign/signature"
+	certkey      = "dev.sigstore.cosign/certificate"
+	chainkey     = "dev.sigstore.cosign/chain"
+	BundleKey    = "dev.sigstore.cosign/bundle"
+	TSABundleKey = "dev.sigstore.cosign/bundle"
```

Comment:
I think you might have copy/pasted BundleKey here!

Reply:
Good catch! I named here as `tsabundle` https://github.com/sigstore/cosign/pull/2454/files#diff-a4859b65c42ca40ac004766c29a8b1279097c2d7909552f77885af38876729c8R36. But I am not good at naming, so we can use 3161timestamp instead.